apps/ecosystem/src/ecosystem.module.ts

```diff
@@ -22,6 +23,6 @@ import { getNatsOptions } from '@credebl/common/nats.config';
     CacheModule.register()
   ],
   controllers: [EcosystemController],
-  providers: [EcosystemService, PrismaService, Logger, EcosystemRepository]
+  providers: [EcosystemService, UserActivityRepository, PrismaService, Logger, EcosystemRepository]
```

Comment:
Any specific reason to add `UserActivityRepository` in the ecosystem module

Reply:
The UserActivityRepository is located in the libs folder. We have added a common function for storing deleted audit trails in the User repository. This function is called from all the microservices to store the deleted audit trails